tests/test_database/charleston/input/projections/current_empty/current_new.toml

```diff
@@ -1,3 +1,5 @@
 name = "current_new"
 long_name = "current_new"
+[physical_projection]
+[socio_economic_change]
```

Comment:
Is this the result from a test to write a projection toml? Doesn't seem to work since it is missing all the keys under physical_projection and socio_economic_change. I cannot see the script with teh Projections class in this review but 
I tested writing out an event and that worked. You can have a look there

Reply:
Or is it because default values are not set to zero? We should do that. That will make it easier iin the FIAT and SFINCS adapters

Reply:
Not really. I actually manually made this file to check what happens if for current conditions the toml file is empty and if the defaults zeros are loaded correctly in the object.